File: 1512. Number of Good Pairs/solution.java
class Solution {
    public int numIdenticalPairs(int[] nums) {
        int n = nums.length ; 
        int rs = 0 ;
        for(int i=0; i<n ; ++i){
            for(int j=i+1 ; j<n ; ++j){
                if( nums[i] == nums[j] ) rs++ ;
            }
        }
        return rs ; 
    }
}

File: 1512. Number of Good Pairs/solution.go
package solution

func numIdenticalPairs(nums []int) int {
    n := len(nums) 
    rs := 0
    f := make( []int , 101 )
    for i:=0 ; i<n ; i++ {
        f[nums[i]]++
    }
    for i:=1 ; i<=100 ; i++ {
        rs += ( f[i] )*( f[i] -1 )/2
    }
    return rs
}